src/components/pages/Keyword.tsx

```diff
@@ -231,6 +232,13 @@ function Keyword() {
                     Googleで☆{shop.rating}
                   </Typography>
                   {/* {shopData.address_components[7].long_name} */}
+                  <Button
+                      component={Link}
+                      to="/post/{shop.id}"
```

Comment:
このリンク先を指定する部分で、
map関数の返り値を反映させたいのですが、どうすればいいのでしょうか。
(/post/の後ろにDBから取得したidを割り当てたい)

Reply:
テンプレートリテラルで変数展開できると思います！
```suggestion
                      to=`/post/${shop.id}`
```